pallets/dao/src/lib.rs

```diff
@@ -107,7 +107,7 @@ pub mod pallet {
 
 	/// Configure the pallet by specifying the parameters and types on which it depends.
 	#[pallet::config]
-	pub trait Config: frame_system::Config + pallet_task::Config {
```

Comment:
Task was injected into Dao, but it was unnecessary since tasks are only represented as Vec<Hash> there. I removed them from the Dao.

Reply:
Okay